evolution.py

```diff
@@ -130,11 +131,13 @@ def evolve(self, T_final, export=True, init_time=True):
         if export:  # and not interrupted:
             self.export()
 
-        return self.data
+#        return self.data
```

Comment:
Not every function needs to return something. Especially when you won't do anything with the returned object.

Reply:
In Python every function returns something — even if it's `None`. I think you're obsessing over irrelevant things with these returns. I added this line because I predicted that one day I might want to do something with `Universe.data` after running a simulation. I don't use it now and I can access it in other ways — that's true, but this change literally does nothing for anybody :)